gitbot.js

```diff
@@ -59,9 +69,15 @@ function findGitRepos(repos, depth, callback) {
  * returns all commits of the last given `days`.
  * Calls `callback` with line-seperated-strings of the formatted commits.
  */
-function getCommitsFromRepos(repos, days, callback) {
+async function getCommitsFromRepos(repos, days, callback) {
   let cmts = [];
-  let gitUser = getGitUser();
+  let gitUser;
+  try {
+    gitUser = await getGitUser();
+  } catch( error ) {
+    callback(error);
```

Comment:
If the error is returned here we propably need error-handling in the function that calls `getCommitsFromRepos`!?

Reply:
I see...so the error is just displayed. I think this is a valid solution! Thx!